Mathlib/Topology/Category/TopCat/Limits/Konig.lean

```diff
@@ -109,8 +108,9 @@ theorem partialSections.closed [∀ j : J, T2Space (F.obj j)] {G : Finset J}
   apply isClosed_biInter
   intro f _
   apply isClosed_eq
-  · continuity
-  · continuity
+  -- `continuity` can solve the two remaining goals but takes about 2s on each.
```

Comment:
Does `fun_prop` work here? (In my experience, if so it is *much* faster.)

Reply:
Great idea, that is indeed so much faster!